doc/source/news.rst

```diff
@@ -7,6 +7,28 @@
 News
 ====
 
+.. _release-7-0-8:
+
+Release 7.0.8 - 2017-10-29
+--------------------------
+
+Improvements
+^^^^^^^^^^^^
+
+* [windows] Supported backtrace on crash.
+  This feature not only function call history but also source filename
+  and number of lines can be displayed as much as possible.
+  This feature makes problem solving easier.
+
+* Supported ``( )`` (empty block) only query (``--query "( )"``) for
+  ``QUERY_NO_SYNTAX_ERROR``. In the previous version, it caused an error.[GitHub#767]
```

Comment:
need space between `.` and `[` ?

Reply:
Thanks, @yoshihara!

I'll fix after merge this PR.